### PR TITLE
IDR web tweaks

### PR DESCRIPTION
**Pull request by ximenesuk**
These are quick fixes that can be done more nicely further down the line.
 + Disable Show Rois (hard-coded)
 + Default to “All users” for search (hard-coded)
 + Map annotation above Table of annotations

### CODE REVIEW

File: components/tools/OmeroWeb/omeroweb/webgateway/templates/webgateway/viewport/omero_image.html [hunk 1]
```diff
@@ -963,11 +963,7 @@ <h3 class="can-collapse" id="bulk-annotations" style="display: none;">Bulk Annot
 {% block roi_buttons %}
           <div id="roi_controls">
                 <h1>ROI Count: {{ roiCount }}</h1>
-                {% ifnotequal roiCount 0 %}
```
Comment (aleksandra-tarkowska) [inline components/tools/OmeroWeb/omeroweb/webgateway/templates/webgateway/viewport/omero_image.html]: Please use operator https://docs.djangoproject.com/en/1.6/ref/templates/builtins/#id3

**Comment by aleksandra-tarkowska** [comment]
@joshmoore searching by ID is https://github.com/ximenesuk/openmicroscopy/blob/idr-minor-web-tweaks/components/tools/OmeroWeb/omeroweb/webclient/views.py#L770

**Comment by aleksandra-tarkowska** [comment]
hold on, but if map annotation contains number both record should be displayed, Image with ID and the other Object that contains that map annotation. If you do not see that, that means map annotation is not found regardless of that code

File: components/tools/OmeroWeb/omeroweb/webgateway/templates/webgateway/viewport/omero_image.html (hunk 1 above)
Comment (ximenesuk) [inline components/tools/OmeroWeb/omeroweb/webgateway/templates/webgateway/viewport/omero_image.html]: I'm happy to fix this if 1000 (or some other number is suitable).

**Comment by joshmoore** [comment]
@aleksandra-tarkowska : agreed. There were other cases though where the ID matches swamped the found map annotation. I'm re-indexing now so we should be able to test.

cc: @ximenesuk for https://github.com/ximenesuk/openmicroscopy/blob/idr-minor-web-tweaks/components/tools/OmeroWeb/omeroweb/webclient/views.py#L770

**Comment by aleksandra-tarkowska** [comment]
Urlizer is your friend https://docs.djangoproject.com/en/1.6/ref/templates/builtins/#urlize

**Comment by aleksandra-tarkowska** [comment]
To add url to map annotation you need to modify  https://github.com/ximenesuk/openmicroscopy/blob/idr-minor-web-tweaks/components/tools/OmeroWeb/omeroweb/webclient/templates/webclient/annotations/mapannotation.html#L59

```
            <td>{{ row.0|urlize|escape }}</td>
            <td>{{ row.1|urlize|escape }}</td>
```
This will only work when map annotation is loaded from the server and template is rendered. Link will not appear when you add value in web UI.

File: components/tools/OmeroWeb/omeroweb/webgateway/templates/webgateway/viewport/omero_image.html (hunk 1 above)
Comment (aleksandra-tarkowska) [inline components/tools/OmeroWeb/omeroweb/webgateway/templates/webgateway/viewport/omero_image.html]: Fixed in https://github.com/openmicroscopy/openmicroscopy/pull/4249/

**Comment by joshmoore** [comment]
Merging for 0.0.3

**Pull request merged**